hack/verify-generate.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Copyright The Shipwright Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Verifies if a developer has forgot to run the
+# `make generate` so that all the changes in the
+# clientset should also be pushed
+
+set -euo pipefail
+
+GOPATH=${GOPATH:-$(go env GOPATH)}
+SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+CODEGEN_PKG=${CODEGEN_PKG:-$(cd ${SCRIPT_ROOT}; ls -d -1 ./vendor/k8s.io/code-generator 2>/dev/null || echo ../../../k8s.io/code-generator)}
```

Comment:
Shellcheck SC2086: Double quote to prevent globbing and word splitting.
```suggestion
CODEGEN_PKG=${CODEGEN_PKG:-$(cd "${SCRIPT_ROOT}"; ls -d -1 ./vendor/k8s.io/code-generator 2>/dev/null || echo ../../../k8s.io/code-generator)}
```